### ISSUE TITLE
WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `addListener` method

### ISSUE THREAD

**Issue by vinayr** [AppsFlyerSDK/appsflyer-react-native-plugin#335]
# Report

## Plugin Version

6.4.0

## On what Platform are you having the issue?

android

## What did you do?

upgrade react-native to latest version 0.66.1

## What did you expect to happen?

should not show any warnings

## What happened instead?

shows this warning:
```
 WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `addListener` method.
 WARN  `new NativeEventEmitter()` was called with a non-null argument without the required `removeListeners` method.
```

**Comment by amit-kremer93** [comment]
@vinayr can you please provide more information about the specific code that causes this warning?

**Comment by vinayr** [comment]
There's no specific code. Just integrate the SDK in the latest version (0.66.1) RN app, and you will see these warnings when app is run.
This might help https://github.com/facebook/react-native/issues/32051

**Comment by professorkolik** [comment]
Any update on this issue?

**Comment by amit-kremer93** [comment]
We will check it and let you know. Thank you for informing us.

**Comment by AdamTyler** [comment]
I am also seeing this issue

**Comment by amit-kremer93** [comment]
This warning will be fixed in the next release. thank you all

**Comment by amit-kremer93** [comment]
issue is fixed in the latest release [6.4.40](https://github.com/AppsFlyerSDK/appsflyer-react-native-plugin/blob/master/CHANGELOG.md#6440)
